emails/sns.py

```diff
@@ -0,0 +1,95 @@
+import base64
+import logging
+import pem
+import six
```

Comment:
we may not need `six` because we don't intend to run on anything except python3?